packages/indexer/src/api/index.ts

```diff
@@ -251,18 +300,31 @@ export async function run(db: DB, logger: Logger) {
     '/tx',
     log,
     async (req: express.Request<{}, {}, {}, MsgRequest>, res) => {
-      const { size } = req.query;
+      const { size: sizeStr } = req.query;
 
-      if (size && size > 30) return fail(res, 403, 'maximum page size is 30');
+      try {
+        if (sizeStr && parseInt(sizeStr) > 30)
```

Comment:
Why is the max size 30? Just curious

Reply:
to prevent system from quering all of the messages at once. Though it is not so useful while we don't have rate limiting and other stuff